Libraries/Settings/Settings.android.js

```diff
@@ -11,24 +11,33 @@
  */
 'use strict';
 
-var Settings = {
```

Comment:
Would it be possible to use the same JS implementation for iOS and Android? You would have to provide the same API for the native module. One thing that the iOS one does is that it tracks subscribers in JS instead of adding a new listener for each `watchKeys` call which I think is better. The main advantage of using the same JS is that we could share that subscription logic.